### PR TITLE
feature/430/unit-test-enenchanced-table/filter-row

### PR DESCRIPTION
**Pull request by FryDay969**

### CODE REVIEW

File: tests/unit/components/enhanced-table/filter-row/filter-row.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import FilterRow from '~/components/enhanced-table/filter-row/FilterRow'
+import { useTableContext } from '~/context/table-context'
+import { vi } from 'vitest'
+import { fireEvent, screen, render, renderWithProviders } from '@testing-library/react'
```
Comment (Yurenko) [inline tests/unit/components/enhanced-table/filter-row/filter-row.spec.jsx:4]: Set the correct order of imports

And name this file correctly 'FilterRow.spec.jsx'

**Review by Yurenko — COMMENTED** [review]
And write test for clear filter
image

**Review by maarekua — APPROVED** [review]

File: tests/unit/components/enhanced-table/filter-row/filter-row.spec.jsx (hunk 1 above)
Comment (FryDay969) [reply tests/unit/components/enhanced-table/filter-row/filter-row.spec.jsx]: Fixed

**Comment by FryDay969** [comment]
> And write test for clear filter image

Fixed

**Review by Yurenko — APPROVED** [review]

**Pull request merged**